proto/aserto/directory/model/v3/model.proto

```diff
@@ -75,87 +75,11 @@ service Model {
         //     }
         // };
     };
-
-    rpc DeleteManifest(DeleteManifestRequest) returns (DeleteManifestResponse) {
```

Comment:
Are we removing Delete? Is the assumption that I have to write a zero byte Body to delete a manifest? I would prefer to keep Delete as an explicit method

Reply:
What does it mean to delete the manifest? Would it delete all objects and relations too?

Reply:
Thinking about the manifest selection and switching experience that needs to be implemented,  how are you going to facilitate that? What happens when I do a second Set with an empty body in your design?

Reply:
I imagined returning an error if trying to delete object or relation types that have instances in the directory.
I didn't think of `SetManifest` as having any destructive side-effects.

Reply:
In that sense, if we want to provide a function that resets the directory to its initial state it might as well be `DeleteManifest`. This way we can keep `SetManifest` "safe" (i.e. doesn't touch the objects or relations tables) and provide an explicit mechanism to reset the directory.
I'll add it back.